amqphandler/amqphandler.go

```diff
@@ -800,24 +806,36 @@ func (handler *AmqpHandler) scheduleMessage(messageType string, data interface{}
 	}
 }
 
+func isMessageImportant(messageType string) bool {
+	for _, importantType := range importantMessages {
+		if messageType == importantType {
+			return true
+		}
+	}
+
+	return false
+}
+
+func getMessageDataForLog(messageType string, data []byte) string {
```

Comment:
what if we make `data` as `string`, so we don't need to cast below in this case? And also this cast isn't cheap
 https://stackoverflow.com/a/14373714

Reply:
After json marchal we have data as []byte to display log properly we have to case it to string. So, the cast is required any case. If I change data to string, it should be casted before call of this function.